s3cortxutils/s3confstore/s3confstore/cortx_s3_confstore.py

```diff
@@ -26,138 +29,166 @@
 from cortx.utils.kv_store import kv_store_collection
 
 class S3CortxConfStore:
-  default_index = "default_cortx_s3_confstore_index"
 
-  def __init__(self):
-    """Instantiate constore"""
+  def __init__(self, config: str = None, index: str = "default_s3confstore_index"):
+    """Instantiate confstore."""
+    self.config_file = config
+    self.default_index = index
 
-  def load_config(self, index: str, backend_url: str):
-    """Load Config into constore"""
-    if not backend_url.strip():
-      print("Cannot load py-utils:confstore as backend_url[{}] is empty.".format(backend_url))
-      exit(-1)
-    else:
-      Conf.load(index, backend_url)
+    if config is not None:
+      self.validate_configfile(config)
+      self.load_config(self.default_index, self.config_file)
+
+  @staticmethod
+  def load_config(index: str = "default_s3confstore_index", config: str = ""):
+    """Load Config into confstore."""
```

Comment:
I see that you've removed the check :
_if not backend_url.strip():_
Is this on purpose?

Reply:
I will add this check in validate function. Thanks for pointing this out

Reply:
done